tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     /* Basic Options */
     "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
-    "module": "system" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
```

Comment:
Without knowing about everything here, is there a rationale behind changing from `system` to `commonjs`? (I have no idea what `system` means).

Reply:
`system` refers to systemjs. 

Using `"module": "system"` causes errors with Angular, and according to [this](https://stackoverflow.com/a/35076930), the solution is to transpile to `commonjs` instead.